### ISSUE TITLE
Many to One Training, Many to Many Testing?

### ISSUE THREAD

**Issue by windfreedom** [keras-team/keras#8731]
I have a weird question. @fchollet 
A friend and I are trying to use LSTMs to predict the expected point value for a basketball possession given coordinate data for all 11 players + the ball.

So, our input data is as follows:
batch_size = 100 posessions
time_steps = 24 seconds per possesion (on the shot clock) * frame rate (number of times the positions of all 11 players + ball are sampled per second) 
features = 33 (x,y,z coordinates for 10 players + ball).

Thus it fits the 3d structure required by Keras: (batch_size, time_steps, features)

The output however, has only ONE point value (0,1,2,3) for each possession so it is: (batch_size, 1) vector. 

So then it seems while training we would set "return_sequence = FALSE" and be on our merry way. 

BUT, after we're done training, what we were hoping to do was to feed a validation set possession into the network and **output** the y-hat value at EACH TIME STEP. Is that even possible? If so, how? 

Thanks :)
